imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,82 @@
+from django.test import TestCase
+import factory
+from . import models
+from .models import Photos, Album
+from django.contrib.auth.models import User
+
+
+class UserFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = models.User
+        django_get_or_create = ('username',)
+    username = 'people'
+
+
+class PhotoFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Photos
+
+    user = UserFactory.create(username='user1')
+
+
+class AlbumFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Album
+
+    user = UserFactory.create(username='user2')
+
+
+class TestPhoto(TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Be cautious.  This method and the work done inside it is **not** rolled back between tests.  Any modification you do to your classes during a test will persist between tests.  This is dangerous.  It's much better to use the `setUp` instance method of the `TestCase` to handle this type of work.